templates/firstboot-nvme.yaml.j2

```diff
@@ -33,6 +33,11 @@ resources:
             if [[ $(hostname) == *$FORMAT* ]] ; then
               sed "s/^\(GRUB_CMDLINE_LINUX=\".*\)\"/\1 $KERNEL_ARGS\"/g" -i /etc/default/grub ;
               grub2-mkconfig -o /etc/grub2.cfg
+              mkdir -p /var/lib/nova
+              mkfs -t xfs -f /dev/nvme0n1
+              mount /dev/nvme0n1 /var/lib/nova
+              chmod a+w /var/lib/nova
```

Comment:
Also you can't do this mounting if you want to passthrough nvme as the device won't be available on the hypervisor. Looks like you need a condition within the template to do this only for the case where you mount nvme.

Reply:
Is the file system xfs even on OSP 13 ? I rememberthat the default is different between OSP 13 and OSP 16, you might want to check.

Reply:
Agree. As we know, NVME passthrough is introduced for OCP. As OCP can't use it (OCP IPI limitation), thought of removing NVME passthrough in later patches.

Reply:
> Is the file system xfs even on OSP 13 ? I rememberthat the default is different between OSP 13 and OSP 16, you might want to check.

Both OSP13 and OSP16.1 are using xfs file systems.